pkg/domain/infra/abi/system.go

```diff
@@ -183,6 +192,58 @@ func (ic *ContainerEngine) SystemPrune(ctx context.Context, options entities.Sys
 		}
 
 		systemPruneReport.ImagePruneReports = append(systemPruneReport.ImagePruneReports, imagePruneReports...)
+
+		// Remove all unused netavark networks.
+		// Does not remove cni networks because it could remove external cni networks.
+		if ic.Libpod.GetNetworkBackend() == "netavark" {
```

Comment:
Do we really want to differ here? As said before system reset already removes all cni networks so I see no reason why prune should not.

Reply:
Thank you for reviewing. I removed network backend branch.
On the other hand, I think there is a option to remove all CNI networks pruning function from "system reset".
If this options is better, I will try to fix "system reset".